packages/dashmate/src/core/wallet/importPrivateKey.js

```diff
@@ -4,10 +4,11 @@
  * @typedef {importPrivateKey}
  * @param {CoreService} coreService
  * @param {string} privateKey
+ * @param {string?} walletName
  * @return {Promise<void>}
  */
-async function importPrivateKey(coreService, privateKey) {
-  return coreService.getRpcClient().importPrivKey(privateKey, { wallet: 'main' });
+async function importPrivateKey(coreService, privateKey, walletName = null) {
```

Comment:
I am not sure about that. We only invoke importPrivateKey() inside `registerMasternodeTask`, which is currently only used during setup local phase. ConfigureCoreTask are creating wallet separately before executing registerMasternodeTask, and its obvious that we should use `main`. But when registerMasternodeTask are invoked via command, it does not have any predefined created wallets, so it will fail with error that it cannot find such wallet.

Should we move createwallet rpc command inside registerMastenodeTask?